.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+# This workflow will create a GitHub release every time a tag is pushed
+name: Create GitHub Release
+
+on:
+  push:
+    tags:
+      - "v2.*"
+      - "v3.*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: "marvinpinto/action-automatic-releases@latest"
```

Comment:
Can we pin this to the current major version so it doesn't fail when they push breaking changes?

Reply:
Afaik you can only set existing release tags. So unfortunately something like _@v1.*_ doesn't work here. I changed it to the latest release v1.2.1, so this can be manually upgraded if necessary.